From f6368ed3946be6a310a83b59f59b7362c02b5717 Mon Sep 17 00:00:00 2001
From: Hugo Osvaldo Barrera <hugo@whynothugo.nl>
Date: Thu, 10 Sep 2026 10:35:22 +0200
Subject: [PATCH] Fix missing property in test helper

Tests fail since they attempt to use this attribute and it is not
defined.
---
 tests/filters/__init__.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/filters/__init__.py b/tests/filters/__init__.py
index 6711b51b..3d782c42 100644
--- a/tests/filters/__init__.py
+++ b/tests/filters/__init__.py
@@ -27,6 +27,7 @@ class Client:
 class User:
     def __init__(self, username: str = None):
         self.username = username
+        self.usernames = []
 
 
 class Message:
-- 
2.55.0

